examples/eco/05-offshore-h2/Nav_HOPP_Model.py

```diff
@@ -142,12 +152,10 @@
 
 
 
-
-
 import numpy as np
-HRly_Op=hopp_results["combined_power_production_hopp"]
+HRly_Op=hopp_results["combined_power_production_hopp"] #Substract fixed power for power grid
 
-HRly_Op2=np.asarray(hopp_results["combined_power_production_hopp"]) #Combined power
+HRly_Op2=np.asarray(hopp_results["combined_power_production_hopp"]) #Combined power for use
```

Comment:
Why save the hope_results output separately in a copy? Is there a reason you don't want to use the dictionary?